votepy/k-borda.py

```diff
@@ -1,8 +1,7 @@
-from typing import Iterable, Sequence, List
 from ordinal_election import OrdinalElection, OrdinalBallot
+from typing import Union, List
```

Comment:
List is deprecated as well for the same reason as before - you can just use the built-in list.

Reply:
Oh, I miss this one. You're right.